src/passes/Asyncify.cpp

```diff
@@ -1115,9 +1115,11 @@ struct AsyncifyLocals : public WalkerPass<PostWalker<AsyncifyLocals>> {
     auto* func = getFunction();
     Index total = 0;
     for (Index i = 0; i < numPreservableLocals; i++) {
-      auto type = func->getLocalType(i);
-      auto size = type.getByteSize();
```

Comment:
can `.getByteSize` of a tuple type return the sum of the sizes? then this part of the patch wouldn't be needed, and might help elsewhere too.

Reply:
I considered this, but decided not to do it because this is the only place that would use it. However, I think the separation of concerns argument is good. I will go all in and add helpers for the load and store patterns as well. Maybe they will be useful elsewhere.

Reply:
I did the change to .getByteSize, but not load or store because there were too many design decisions to make about alignment and offsets and things that it didn't seem worth it after all with just a single use.